src/bundle/ParamConverter/ContentTypeParamConverter.php

```diff
@@ -52,7 +53,11 @@ public function apply(Request $request, ParamConverter $configuration)
 
         if ($request->get(self::PARAMETER_CONTENT_TYPE_ID)) {
             $id = (int)$request->get(self::PARAMETER_CONTENT_TYPE_ID);
-            $contentType = $this->contentTypeService->loadContentType($id, $prioritizedLanguages);
+            try {
+                $contentType = $this->contentTypeService->loadContentType($id, $prioritizedLanguages);
+            } catch (NotFoundException $e) {
+                $contentType = $this->contentTypeService->loadContentTypeDraft($id);
```

Comment:
Should it always load draft (on exception) for all contentTypeId / contentTypeIdentifier placeholders? It's used in different places, so is it ok to inject the draft content in some older endpoints?